rpc-server/src/modules/queries/mod.rs

```diff
@@ -57,7 +57,10 @@ impl near_vm_logic::External for CodeStorage {
         ))
     }
 
-    #[cfg_attr(feature = "tracing-instrumentation", tracing::instrument(skip(self)))]
+    #[cfg_attr(
+        feature = "tracing-instrumentation",
+        tracing::instrument(skip(self, _mode))
```

Comment:
I had to skip `_mode` as well since it's not derives the Debug (IIRC the cargo check error)